test/integration-tests/commands/build.test.ts

```diff
@@ -43,10 +39,11 @@ suite("Build Commands", function () {
 
     activateExtensionForSuite({
         async setup(ctx) {
-            // The description of this package is crashing on Windows with Swift 5.9.x and below,
-            // preventing it from being built. The cleanup in the teardown is failng as well with
-            // an EBUSY error. Skip this test on Windows until the issue is resolved.
-            if (process.platform === "win32") {
+            // The description of this package is crashing on Windows with Swift 5.9.x and below
```

Comment:
This comment says 5.9 but code indicates 6.0

Reply:
Incoming PR I'll fix that in, but thanks good catch